libs/packages/layouts/src/lib/feature/system-alert/system-alert.component.html

```diff
@@ -56,15 +56,15 @@
   <div *ngFor="let alert of alerts; let i = index;" class="grid-col-12" [attr.id]="'alert' + i">
     <div class="sds-alert--header__content grid-row" id="sds-alert--header__content">
       <div class="grid-col-fill">
-        <div class="sds-alert--header__label margin-right-1">
-          {{alert.header}}
-        </div>
-        <a (click)="onShowDetailsClicked(alert)" href="javascript:void(0)" class="sds-alert--header__link">
-          Show Details
+        <div class="sds-alert--header__label margin-right-1" [innerHTML]="alert.header"></div>
```

Comment:
Also updated header to be innerHTML rather than plain text to allow anchor tags in header.